indico/modules/events/timetable/controllers/manage.py

```diff
@@ -238,10 +238,8 @@ def _change_parent(self, session_block_id):
             self.break_.session = None
             if entry.parent is not None:
                 update_timetable_entry(entry, {'parent': None, 'start_dt': entry.start_dt})
-        try:
-            target_id = int(session_block_id)
-        except (TypeError, ValueError):
-            raise BadRequest('Invalid session block id !!!!')
+
+        target_id = int(session_block_id)
```

Comment:
You still need to return early from this method in case of a null `session_block_id` - and then the `int()` isn't needed anymore because it will always be an int (so you can just use `session_block_id` instead of `target_id` below).

Right now things still break if you move a break from a session block to the top level.